lib/src/api/tolgee_api.dart

```diff
@@ -31,17 +31,18 @@ class TolgeeApi {
   }
 
   /// Gets all translations in Tolgee project
-  static Future<TolgeeTranslationsResponse> getTranslations({
-    required TolgeeConfig config,
-  }) async {
+  static Future<TolgeeTranslationsResponse> getTranslations(
+      {required TolgeeConfig config,
+      required Map<String, TolgeeProjectLanguage> projectLanguages}) async {
     List<TolgeeKeyModel> allTranslations = [];
+    String languages = projectLanguages.keys.join(',');
     int currentPage = 0;
     int totalPages = 1; // Initialize to 1 to enter the loop
 
     while (currentPage < totalPages) {
       final response = await get(
         Uri.parse(
-            '${config.apiUrl}/projects/translations?page=$currentPage&size=20&sort=keyId,asc'),
+            '${config.apiUrl}/projects/translations?page=$currentPage&size=20&sort=keyId,asc&languages=$languages'),
         headers: {
```

Comment:
Will you always fetch all languages, or only the one selected? Because I think if you always fetch all languages, the response size could potentially get too big in the future

Reply:
Whoops, I will fix it :)